ignitionrobotics/ign-transport
# Configurable IP address and port for discovery

See issue #192.

How to test it:

Run the `example/publisher` with:

```
IGN_DISCOVERY_MULTICAST_IP=239.255.0.7 IGN_DISCOVERY_MSG_PORT=11000 IGN_DISCOVERY_SRV_PORT=11001 IGN_VERBOSE=1 ./publisher
```

And the `example/subscriber` with:

```
IGN_DISCOVERY_MSG_PORT=11000 IGN_DISCOVERY_SRV_PORT=11001 ./subscriber
```

You should receive the messages as expected. Then, run:

```
./subscriber
```

And you shouldn't get any messages.
